Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1378,20 +1378,6 @@ CanvasFiltersEnabled:
     WebCore:
       default: false
 
-CanvasNoiseInjectionEnabled:
```

Comment:
I would've left it, but it's probably safe to delete at this point.

Reply:
Ah. Yeah, I think it's definitely safe to remove at this point, now that it's already (mostly) stable in ToT.